### DISCUSSION TITLE
Version 1.0.1 or 1.1.0?

### DISCUSSION THREAD

**Discussion by tig** [tui-cs/Terminal.Gui#1280] [General]
The following have been merged with `main` since we launched 1.0.0:

These are purely bug fixes that have no impact on backwards compat.
- https://github.com/migueldeicaza/gui.cs/pull/1278 - Fixes #1272 - Linux/Mac window sizing
- https://github.com/migueldeicaza/gui.cs/pull/1274 - Fixes #1273 - Add direction summaries

These, arguably, add/change functionality, but are backwards compatible. 
- https://github.com/migueldeicaza/gui.cs/pull/1277- Fixes #1276 - Added TextDirection constructor to View and Label and improving AutoSize
- https://github.com/migueldeicaza/gui.cs/pull/1271 - TextView CursorLeft/CursorRight change focus when at start/end 

Thus, based on the 'rules' of SemVer (https://semver.org/) I believe the next version should be 1.1.0 Correct?

![image](https://user-images.githubusercontent.com/585482/116939007-73671b00-ac20-11eb-97b7-e8acec689e22.png)

**Comment by BDisp** [comment]
I think it should be `1.0.1` because there only  backwards compatible bug fixes.

**Reply by tig** [reply]
> I think it should be `1.0.1` because there only backwards compatible bug fixes.

I don't think so. #1277 literally adds a new feature (TextDirection to constructor). #1271 literally changes behavior. Right?

(I think debating this is important because I want to ensure we're all aligned).

**Reply by BDisp** [reply]
Yes I agree with you too.

**Comment by tig** [comment]
I've pushed v1.1.0. 

However, Action workflows are not running for some reason.

**Comment by tig** [comment]
Well, that explains that:

![image](https://user-images.githubusercontent.com/585482/118401402-6849bd00-b61a-11eb-84ef-85551cd68f94.png)

**Reply by BDisp** [reply]
Just wait until they fix it :-)
